src/typescript/frontend/src/components/pages/emojicoin/components/main-info/MainInfo.tsx

```diff
@@ -192,21 +222,19 @@ const MainInfo = ({ data }: MainInfoProps) => {
         </div>
 
         <div className={`flex flex-col justify-between ${borderStyle}`}>
+          <div className="flex justify-between">
+            <div className={statsTextClasses + " text-light-gray"}>
+              APT&nbsp;
+              <AptosIconBlack className={"icon-inline mb-[0.0ch]"} />
+              &nbsp;/&nbsp;USD&nbsp;$:
```

Comment:
```suggestion
              {"APT "}
              <AptosIconBlack className={"icon-inline mb-[0.0ch]"} />
              {" / USD $:"}
```

Think I got it right, but yea just to make it clearer